tests/test_views.py

```diff
@@ -0,0 +1,82 @@
+import subjects
+from tests import factories
+from django.urls import reverse
+import pytest
+import os
+
+
+@pytest.mark.django_db
+class TestSubjectViews:
```

Comment:
So I think we can improve the structure of this test file and test_admin_views.py if we use smaller test classes, with one per view, like the tests here: https://github.com/unt-libraries/django-nomination/blob/master/tests/test_views.py. Alternately, we could just do away with the class and make the tests functions rather than methods.